packages/core/src/interpreter.ts

```diff
@@ -252,7 +259,8 @@ export class Interpreter<
 
     // Update children
     this.children.forEach((child) => {
-      this.state.children[child.id] = child;
+      const name = 'id' in child ? child.id : '';
```

Comment:
as a child has to be `SpawnedActorRef` now the `'id' in child` check is not required now